api/integrations/launch_darkly/services.py

```diff
@@ -109,15 +119,199 @@ def _create_tags_from_ld(
     return tags_by_ld_tag
 
 
+# Based on: https://docs.launchdarkly.com/sdk/concepts/flag-evaluation-rules#operators
+def _ld_operator_to_flagsmith_operator(ld_operator: str) -> Optional[str]:
+    return {
+        "in": constants.IN,
+        "endsWith": constants.REGEX,
+        "startsWith": constants.REGEX,
+        "matches": constants.REGEX,
+        "contains": constants.CONTAINS,
+        "lessThan": constants.LESS_THAN,
+        "lessThanOrEqual": constants.LESS_THAN_INCLUSIVE,
+        "greaterThan": constants.GREATER_THAN,
+        "greaterThanOrEqual": constants.GREATER_THAN_INCLUSIVE,
+        "before": constants.LESS_THAN,
+        "after": constants.GREATER_THAN,
+        "semVerEqual": constants.EQUAL,
+        "semVerLessThan": constants.LESS_THAN,
+        "semVerGreaterThan": constants.GREATER_THAN,
```

Comment:
Does flagsmith automatically understand those are versions and compares them according to the semver conventions?

I saw `SemVer` stuff on UI but it seems like they are just regular gt/lt in the backend.

Reply:
no - flagsmith does not try and interpret string values with gt/lt - we have specific semver-aware operators that are in addiotn to gt/lt

Reply:
Seems like there is a conflicting view, we can go over this together Wednesday's meeting.

Reply:
You need to add a `:semver` suffix to the segment value. This will prompt the engine to use the correct matching function as seen here:

[flag_engine/utils/semver.py
](https://github.com/Flagsmith/flagsmith-engine/blob/2f09b52be8fc8a872a43cd4732b41f769e487fcb/flag_engine/utils/semver.py#L15-L16)